lisa/tools/journalctl.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Journalctl(Tool):
+    @property
+    def command(self) -> str:
+        return "journalctl"
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def logs_for_unit(self, unit_name: str) -> str:
+        result = self.run(
```

Comment:
This may generate a lot of log in LISA and flood other logs. Save the output to the remote working path of tool, and copy it back.

Reply:
You can pass in the test case local working path, so the copy back can be written in this tool for convenient.

Reply:
I believe an alternative might be to pass `no_info_log=True` and `no_debug_log=True` here.

Reply:
no_info_log is True by default. Added `no_debug_log=True`.